backend/controllers/cellMove/selectCell.ts

```diff
@@ -74,7 +79,7 @@ const getCellOccupants = async (res, { prisonApi, activeCaseLoadId, cells, nonAs
             nonAssociations.nonAssociations &&
             nonAssociations.nonAssociations.find((na) => na.offenderNonAssociation.offenderNo === occupant.offenderNo)
         ),
-        csra: csraInfo && csraInfo.classification,
+        csra: (csraInfo && translateCsra(csraInfo.classificationCode)) || 'Not entered',
```

Comment:
Not sure this fixes the problem?

Reply:
See other response, I assume it's the same idea